.github/workflows/build.yaml

```diff
@@ -25,10 +25,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Use node 20
+      - name: Use node 22
```

Comment:
We'll want to update the `.nvmrc` to be node 22 now, so CI and local are aligned. Without node 22, I also ran into:

```
│ > @linear/sdk@65.2.0 update:scalars /Users/steven/repos/linear/packages/sdk
│ > node scripts/update-scalars.ts
│ node:internal/modules/esm/get_format:219
│   throw new ERR_UNKNOWN_FILE_EXTENSION(ext, filepath);
│         ^
│ TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/steven/repos/linear/packages/sdk/scripts/update-scalars.ts
```